database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -16,6 +16,7 @@ public function up()
         Schema::create('users', function (Blueprint $table) {
             $table->bigIncrements('id');
             $table->foreignId('track_id')->nullable()->constrained()->onDelete('cascade');
+            $table->foreignId('team_id')->nullable();
```

Comment:
What happens if the team gets deleted? Shouldn't we add a constraint that sets this to null if so?